### DISCUSSION TITLE
Vue3 support

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [biomejs/biome#116] [Ideas]
As Rome was never able to get around to adding support for Vue3 or other JS frameworks would be great!

**Comment by ematipico** [comment]
Closing. We have plans, but first we have to parse CSS and HTML. https://github.com/biomejs/biome/pull/238

**Comment by ThatOneCalculator** [comment]
@ematipico can this be revisited?

**Reply by dyc3** [reply]
This is actively being worked on. We have partial support for frameworks like vue and svelte, but we only parse and lint the javascript sections.

**Reply by ThatOneCalculator** [reply]
Given CSS is already working in biome, would it be too much trouble to get `<style>` portions of Vue files formatted?

**Reply by ematipico** [reply]
PRs are welcome

**Reply by DavidVaness** [reply]
> This is actively being worked on. We have partial support for frameworks like vue and svelte, but we only parse and lint the javascript sections.

Awesome to hear! Cant wait to try this out in our monorepo

**Reply by orimay** [reply]
Would it be too hard to implement `prettier.vueIndentScriptAndStyle` alternative? It's nonsense, that we have in-HTML, in-CSS and in-JS indentation, but no indentation inside script and style tags, as if they were the same level statements, as the content they wrap
